.github/workflows/forge-unstable.yaml

```diff
@@ -133,3 +133,14 @@ jobs:
       COMMENT_HEADER: forge-indexer
       FORGE_TEST_SUITE: indexer_test
       FORGE_ENABLE_INDEXER: true
+  forge-indexer-sdk:
+    if: ${{ github.event_name != 'pull_request' }}
+    needs: determine-test-metadata
+    uses: aptos-labs/aptos-core/.github/workflows/workflow-run-forge.yaml@main
+    secrets: inherit
+    with:
+      IMAGE_TAG: ${{ needs.determine-test-metadata.outputs.IMAGE_TAG }}
+      FORGE_NAMESPACE: forge-indexer-sdk-test-${{ needs.determine-test-metadata.outputs.BRANCH_HASH }}
+      COMMENT_HEADER: forge-indexer-sdk
+      FORGE_TEST_SUITE: forge_indexer_sdk
```

Comment:
where is this test suite defined?

Reply:
iiuc what you're trying to do, you shoud:
* `FORGE_TEST_SUITE: indexer_test`
* `FORGE_DEPLOYER_PROFILE: forge_indexer_sdk`